ethcore/src/engines/mod.rs

```diff
@@ -24,28 +24,30 @@ mod signer;
 mod validator_set;
 
 pub mod block_reward;
-pub mod epoch;
 
 pub use self::authority_round::AuthorityRound;
 pub use self::basic_authority::BasicAuthority;
 pub use self::epoch::{EpochVerifier, Transition as EpochTransition};
 pub use self::instant_seal::{InstantSeal, InstantSealParams};
 pub use self::null_engine::NullEngine;
 
+// TODO [ToDr] Remove re-export
```

Comment:
It would be really helpful for others to pick up this TODO if you can create a new issue for this and refer the issue number here. And I think we should do this for any newly added TODOs!